templates/topic/topic-en.html

```diff
@@ -1,95 +1,37 @@
-			</div>
-			<div class="col-md-6">
-				<h3><a href="#">[Subtopic hyperlink text]</a></h3>
-				<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
-			</div>
-			<div class="col-md-6">
-				<h3><a href="#">[Subtopic hyperlink text]</a></h3>
-				<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
-			</div>
-			<div class="col-md-6">
-				<h3><a href="#">[Subtopic hyperlink text]</a></h3>
-				<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
-			</div>
-			<div class="col-md-6">
-				<h3><a href="#">[Subtopic hyperlink text]</a></h3>
-				<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
-			</div>
-			<div class="col-md-6">
-				<h3><a href="#">[Subtopic hyperlink text]</a></h3>
-				<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
-			</div>
-			<div class="col-md-6">
-				<h3><a href="#">[Subtopic hyperlink text]</a></h3>
-				<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
-			</div>
-			<div class="col-md-6">
-				<h3><a href="#">[Subtopic hyperlink text]</a></h3>
-				<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
-			</div>
-			<div class="col-md-6">
-				<h3><a href="#">[Subtopic hyperlink text]</a></h3>
-				<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
-			</div>
-			<div class="col-md-6">
-				<h3><a href="#">[Subtopic hyperlink text]</a></h3>
-				<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
-			</div>
-			<div class="col-md-6">
-				<h3><a href="#">[Subtopic hyperlink text]</a></h3>
-				<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
-			</div>
+
+{%- include /components/gc-most-requested/gc-most-requested.html -%}
+
+<div class="container">
+	{%- include /components/gc-srvinfo/3col-hidden-title.html -%}
+
+	<div class="row mrgn-tp-xl">
+		<div class="col-md-8">
+			{%- include /components/gc-features/one-tile-hidden-title.html -%}
+		</div>
+		<div class="col-md-4">
+			{%- include gc-follow-us/gc-follow-us.html -%}
```

Comment:
Take a look at that include, it use a popup for when there is multiple account (see facebook) and now that pattern has changed via the "more" last link in that list.